site/e2e/helpers.ts

```diff
@@ -0,0 +1,24 @@
+import { Page } from "@playwright/test"
+
+export const buttons = {
+  starterTemplates: "Starter templates",
+  dockerTemplate: "Develop in Docker",
+  useTemplate: "Use template",
+  createTemplate: "Create template",
+  createWorkspace: "Create workspace",
+  submitCreateWorkspace: "Create workspace",
+  stopWorkspace: "Stop",
+  startWorkspace: "Start",
+}
+
+export const clickButton = async (page: Page, name: string): Promise<void> => {
+  await page.getByRole("button", { name, exact: true }).click()
+}
+
+export const fillInput = async (
+  page: Page,
+  label: string,
+  value: string,
+): Promise<void> => {
+  await page.fill(`text=${label}`, value)
+}
```

Comment:
Is there a reason we made helpers for these? They seem to wrap fairly trivial built-in methods.

Reply:
To address this comment and the one about the constant: previously, we wrote Playwright tests with POMs, so for each page, you'd have a file with a class and methods. I started doing that here and it felt too heavy since the current plan isn't to write many e2e tests. (Although after seeing how pleasant they were to debug, I'm reconsidering that!) So instead, I just dumped everything I would normally have put in the class into these few helpers. But I agree that it's not *that* much easier to read and write than just inlining everything.

Reply:
As someone who isn't familiar with playwright, I appreciate them and makes test-authoring seem a bit easier. No strong opinions though!